## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 475394`
Commit: `73ec78bd82b6bffd1a552637823cb1d7c7e29fcd`

---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>iaito</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 13 packages built:</summary>
  <ul>
    <li>apkleaks</li>
    <li>apkleaks.dist</li>
    <li>diffoscope</li>
    <li>diffoscope.dist</li>
    <li>diffoscope.man</li>
    <li>jadx</li>
    <li>python313Packages.r2pipe</li>
    <li>python313Packages.r2pipe.dist</li>
    <li>python314Packages.r2pipe</li>
    <li>python314Packages.r2pipe.dist</li>
    <li>quark-engine</li>
    <li>quark-engine.dist</li>
    <li>radare2</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>iaito</summary>
<pre>../core/Iaito.cpp: In member function &#x27;QList&lt;AnalVTableDescription&gt; IaitoCore::getAnalClassVTables(const QString&amp;)&#x27;:
../core/Iaito.cpp:3571:5: error: &#x27;RVector&#x27; was not declared in this scope; did you mean &#x27;QVector&#x27;?
 3571 |     RVector *vtables = r_anal_class_vtable_get_all(core-&gt;anal, cls.toUtf8().constData());
      |     ^~~~~~~
      |     QVector
../core/Iaito.cpp:3571:14: error: &#x27;vtables&#x27; was not declared in this scope
 3571 |     RVector *vtables = r_anal_class_vtable_get_all(core-&gt;anal, cls.toUtf8().constData());
      |              ^~~~~~~
../core/Iaito.cpp:3586:5: error: &#x27;r_vector_free&#x27; was not declared in this scope; did you mean &#x27;r_event_free&#x27;?
 3586 |     r_vector_free(vtables);
      |     ^~~~~~~~~~~~~
      |     r_event_free
[435/596] Compiling C++ object iaito.p/Main.cpp.o
[436/596] Compiling C++ object iaito.p/common_RichTextPainter.cpp.o
[437/596] Compiling C++ object iaito.p/widgets_OverviewView.cpp.o
[438/596] Compiling C++ object iaito.p/dialogs_AboutDialog.cpp.o
[439/596] Compiling C++ object iaito.p/widgets_DisassemblerGraphView.cpp.o
[440/596] Compiling C++ object iaito.p/dialogs_CommentsDialog.cpp.o
[441/596] Compiling C++ object iaito.p/dialogs_InitialOptionsDialog.cpp.o
ninja: build stopped: subcommand failed.</pre>
</details>
</details>

--- a/pkgs/development/tools/analysis/radare2/default.nix
+++ b/pkgs/development/tools/analysis/radare2/default.nix
@@ -32,33 +32,33 @@ let
   binaryninja = fetchFromGitHub {
     owner = "Vector35";
     repo = "binaryninja-api";
-    rev = "c40a5f04deec68d388b2072dc42b29141089f9ce"; # https://github.com/radareorg/radare2/blob/master/subprojects/binaryninja.wrap
-    hash = "sha256-IfuGgwVI51urQxhaYkYsE45NkScgxKmmEBV6Pllhwmo=";
+    rev = "5accfca2dc53fe5685dbe9e74f519a7f2f657d48"; # https://github.com/radareorg/radare2/blob/master/subprojects/binaryninja.wrap
+    hash = "sha256-ysTLRmD372mZNAxL+ua7tsPsYyFVPaiDctLUNKKIE4o=";
   };
 
   sdb = fetchFromGitHub {
-    owner = "radare";
+    owner = "radareorg";
     repo = "sdb";
-    tag = "2.2.0"; # https://github.com/radareorg/radare2/blob/master/subprojects/sdb.wrap
-    hash = "sha256-S/aL3F6+Z/rqelfIJaZaBF1IxSmhA1qE9ahFvKARoaE=";
+    tag = "2.3.0"; # https://github.com/radareorg/radare2/blob/master/subprojects/sdb.wrap
+    hash = "sha256-u8sD278Gd5XZkxJZikojh44WketQImx+FtHBQEj8EO0=";
   };
 
   qjs = fetchFromGitHub {
     owner = "quickjs-ng";
     repo = "quickjs";
-    rev = "7238ee64dbc2fbdea044555cda8cda78785a93ed"; # https://github.com/radareorg/radare2/blob/master/subprojects/qjs.wrap
-    hash = "sha256-1ZeLCTmbrlRrZB9El3L497gt3QUA5GIScrFVIBkxA88=";
+    rev = "d405777f7eefa22c17c12970317ef3d6e7658f5a"; # https://github.com/radareorg/radare2/blob/master/subprojects/qjs.wrap
+    hash = "sha256-6/9DmEU+CrZiykiKdLaQy7ds1TvNuyEbat7M6RNuTgs=";
   };
 in
 stdenv.mkDerivation (finalAttrs: {
   pname = "radare2";
-  version = "6.0.4";
+  version = "6.0.7";
 
   src = fetchFromGitHub {
     owner = "radare";
     repo = "radare2";
     tag = finalAttrs.version;
-    hash = "sha256-B0xCQPD27NcCDvRJK1/yyjtrlEJoSTD7e2cu5JIo95w=";
+    hash = "sha256-/WVzqHaDwztvLskE9mSewPlfnVfFPRHQYKvuOitSeNo=";
   };
 
   mesonFlags = [

